### PR TITLE
Fixed issue #2, Add issue template: bug report

### PR DESCRIPTION
**Pull request by rithgan**
# Related Issue
issue #2 

# Proposed Changes
Add issue template: bug report

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+A clear and concise description of what the bug is.
+
+**Steps To Reproduce the behaviour -**
+Steps in points to reproduce the bug
+
+**Expected behaviour -**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or screen recordings -**
+If applicable, add screenshots/ screen recordings to help explain your problem.
```
Comment (ShauryaAg) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: Please remove the space after `/`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**System Information(please complete the following information):**
```
Comment (ShauryaAg) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: Please add a space before `(`

**Review by ShauryaAg — CHANGES_REQUESTED** [review]
Please add **Logs (if applicable)** too

**Comment by rithgan** [comment]
Please review.

Also my discord username is - kiraa#7281

**Review by ShauryaAg — APPROVED** [review]

**Comment by rithgan** [comment]
@ShauryaAg I am really sorry but I think I accidentally closed this pull request. How can I reopen it?

**Comment by ShauryaAg** [comment]
@rithgan you also force pushed on your branch, making the previous changes invalid. Please revert the last commit, and then I can reopen the PR for you again

**Comment by rithgan** [comment]
> @rithgan you also force pushed on your branch, making the previous changes invalid. Please revert the last commit, and then I can reopen the PR for you again

Thanks but I think I got it. I had rebased origin/main to upstream/main due to which I went to the last commit on the original branch. All I had to was make the commit again on the forked branch and reopen the pull request.

**Comment by adityabisoi** [comment]
@rithgan Also remember to use auto issue closing writing syntax, such as closes #2 https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

**Pull request merged**